youtube_dl/extractor/egghead.py

```diff
@@ -82,3 +79,29 @@ def _real_extract(self, url):
             'view_count': int_or_none(lesson.get('plays_count')),
             'tags': try_get(lesson, lambda x: x['tag_list'], list),
         }
+
+        def find_id_and_dlurl():
+            vid_id = lesson.get('wistia_id')
```

Comment:
This has been removed entirely, we aren't using Wistia for streaming any longer.